File: VB/ShowLabelsOnHotTrack/Form1.vb
#Region "#usings"
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraCharts
' ...
#End Region ' #usings

Namespace ShowLabelsOnHotTrack
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        #Region "#code"
        Private m_HotTrackedPoint As SeriesPoint

        Private Sub chartControl1_ObjectHotTracked(ByVal sender As Object, ByVal e As HotTrackEventArgs) Handles chartControl1.ObjectHotTracked
            Dim point As SeriesPoint = TryCast(e.AdditionalObject, SeriesPoint)
            If Not Object.ReferenceEquals(point, m_HotTrackedPoint) Then
                m_HotTrackedPoint = point
                chartControl1.Refresh()
            End If
        End Sub

        Private Sub chartControl1_CustomDrawSeriesPoint(ByVal sender As Object, ByVal e As CustomDrawSeriesPointEventArgs) Handles chartControl1.CustomDrawSeriesPoint

            If Not Object.Equals(e.SeriesPoint, m_HotTrackedPoint) Then
                e.LabelText = ""
            End If
        End Sub
        #End Region ' #code

    End Class
End Namespace

File: CS/ShowLabelsOnHotTrack/Form1.cs
#region #usings
using System;
using System.Windows.Forms;
using DevExpress.XtraCharts;
// ...
#endregion #usings

namespace ShowLabelsOnHotTrack {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

#region #code
SeriesPoint m_HotTrackedPoint;

void chartControl1_ObjectHotTracked(object sender, HotTrackEventArgs e) {
    SeriesPoint point = e.AdditionalObject as SeriesPoint;
    if (!Object.ReferenceEquals(point, m_HotTrackedPoint)) {
        m_HotTrackedPoint = point;
        chartControl1.Refresh();
    }
}

void chartControl1_CustomDrawSeriesPoint(object sender, CustomDrawSeriesPointEventArgs e) {
    e.LabelText = "";
    if (e.SeriesPoint == m_HotTrackedPoint) {
        e.LabelText = e.SeriesPoint.Values[0].ToString();
    }
}
#endregion #code

    }
}
